src/test/java/com/github/dockerjava/api/model/BindTest.java

```diff
@@ -62,4 +98,19 @@ public void toStringDefaultAccessMode() {
         assertEquals(Bind.parse("/host:/container").toString(), "/host:/container:rw");
     }
 
+    @Test
+    public void toStringReadOnlySEL() {
+        assertEquals(Bind.parse("/host:/container:ro,Z").toString(), "/host:/container:ro,Z");
```

Comment:
BindTest.toStringReadOnlySEL:103 expected:</host:/container:ro,Z> but was:</host:/container:ro,Z1>

makes no big difference to

 java.lang.AssertionError:

Expected: is "/host:/container:ro,Z1"
     but: was "/host:/container:ro,Z"
	at com.github.dockerjava.api.model.BindTest.toStringReadOnlySEL(BindTest.java:105)
`
and both look pretty readable. 

AssertThat(..., is(equalTo())) constructs are usually named domain specific language, so DSL.